queue_services/entity-bn/src/entity_bn/worker.py

```diff
@@ -73,6 +73,7 @@ async def process_event(msg: Dict, flask_app: Flask):  # pylint: disable=too-man
     with flask_app.app_context():
         if msg['type'] == 'bc.registry.admin.bn':
             await admin.process(msg)
+            return
```

Comment:
What is this a fix fox? Please call it out in your PR description or ticket so it's recorded.

Reply:
This return was removed by mistake (adding it back now). It was not breaking any functionality but its adding unwanted logs in entity-bn.